### DISCUSSION TITLE
How to fix history?

### DISCUSSION THREAD

**Discussion by PaperOrb** [FreeTubeApp/FreeTube#3212] [Q&A]
Hi, I'm on 18.0 beta appimage for linux, and for the last 2 months, my history has been gone. Also any new videos I watch do not get added to the history either. Any idea what the cause might be?

edit: I moved the freetube configs from .config onto the desktop, reopened the program, and it seems to be adding new videos to history now. Not a very satisfying answer though since I don't know how to recover all of the history I lost.

**Comment by PaperOrb** [comment]
Just an update to say this issue has now happened to my saved videos as well... ended up losing hundreds of saved videos.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @PaperOrb apologies for the delayed response. Would u be willing to provide the affected database files. We would like to investigate what caused this issue. U can find the files [here](https://docs.freetubeapp.io/usage/data-location/).

**Reply by PaperOrb** [reply]
Sorry, I wouldn't feel comfortable sharing that. And in any case, I deleted the whole freetube config folder and appimage, redownloaded it, and imported from an old backup.
